include/odp/api/spec/ipsec.h

```diff
@@ -262,6 +263,14 @@ typedef struct odp_ipsec_capability_t {
 	/** Supported authentication algorithms */
 	odp_crypto_auth_algos_t   auths;
 
+	/** Support of traffic manager (TM) after inline IPSEC processing. On
```

Comment:
It could be mentioned that this is about outbound IPsec only: i.e. "...after outbound inline IPsec processing"

Reply:
Agreed. Will fix this as part of next version.